### ISSUE TITLE
'github-backup' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by evandrocoan** [josegonzalez/python-github-backup#112]
How to run this on Windows?
```
D:\GitHubBackup>github-backup
'github-backup' is not recognized as an internal or external command,
operable program or batch file.
```

I installed it with:
```
D:\GitHubBackup>python --version
Python 3.7.2

D:\GitHubBackup>pip --version
pip 19.1.1 from F:\Python\lib\site-packages\pip (python 3.7)

D:\GitHubBackup>pip install github-backup
Collecting github-backup
Installing collected packages: github-backup
Successfully installed github-backup-0.22.2

D:\GitHubBackup>
```

I also tried:
```
D:\GitHubBackup>F:\Python\Scripts\github-backup.exe
This version of F:\Python\Scripts\github-backup.exe is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher.

D:\GitHubBackup>
```

![image](https://user-images.githubusercontent.com/5332158/58845775-d9bd1a80-8652-11e9-9cee-c80ce541aec8.png)

**Comment by josegonzalez** [comment]
I don't use windows, so you'll need to find someone who has run scripts on windows before to figure this out. Sorry.

**Comment by evandrocoan** [comment]
I manage to run and install it on https://github.com/microsoft/WSL and using its custom shell to run the application.

**Comment by einsteinx2** [comment]
Yep WSL is probably the best option. It would also probably be possible to use something like MSYS2 (https://www.msys2.org/) or Cygwin (https://www.cygwin.com/) if you're not on Windows 10 and so WSL is not available.

**Comment by evandrocoan** [comment]
@einsteinx2, I tried using Cygwin and it did not worked. But I did not tried using `msys2`, if someone managed to use `msys2`, share here if it worked or not.

**Comment by einsteinx2** [comment]
If you can use WSL I'd argue that's better as I think it will eventually more or less replace cygwin and msys2 completely going forward. I only mentioned it for users on Windows 7 or 8. I don't use Windows at all, so I have no intention of trying it myself, but maybe someone else will.

**Comment by jonadem** [comment]
I managed to make it work on windows.

`pip install github-backup`  as in the README

After that I download the file here : https://github.com/josegonzalez/python-github-backup/blob/master/bin/github-backup

Execute it in the local folder : `python .\github-backup`
